app/src/main/java/org/jellyfin/androidtv/util/profile/deviceProfile.kt

```diff
@@ -280,13 +280,49 @@ fun createDeviceProfile(
 		}
 	}
 
-	// Limit video resolution support for older devices
-	codecProfile {
-		type = CodecType.VIDEO
+	// Get max resolutions for common codecs
+	// AVC
+	if(supportsAVC){
+		val maxResolutionAVC = mediaTest.getMaxResolution(MediaFormat.MIMETYPE_VIDEO_AVC)
```

Comment:
Move these variables up to where the maxResolution was before

Reply:
done